src/AppContainer.dev.js

```diff
@@ -54,38 +54,46 @@ class AppContainer extends Component {
 
     if (this.props.component) {
       return <this.props.component {...this.props.props} />;
-    } else {
-      return React.Children.only(this.props.children);
     }
+
+    return React.Children.only(this.props.children);
   }
 }
 
 AppContainer.propTypes = {
-  component: function (props) {
+  component(props) {
     if (props.component) {
       return new Error(
-        `Passing "component" prop to <AppContainer /> is deprecated. ` +
-        `Replace <AppContainer component={App} /> with <AppContainer><App /></AppContainer>.`
+        'Passing "component" prop to <AppContainer /> is deprecated. ' +
+        'Replace <AppContainer component={App} /> with <AppContainer><App /></AppContainer>.'
       );
     }
+
+    return undefined;
   },
-  props: function (props) {
+  props(props) {
     if (props.props) {
       return new Error(
-        `Passing "props" prop to <AppContainer /> is deprecated. ` +
-        `Replace <AppContainer component={App} props={{ myProp: myValue }} /> with <AppContainer><App myProp={myValue} /></AppContainer>.`
+        'Passing "props" prop to <AppContainer /> is deprecated. ' +
+        'Replace <AppContainer component={App} props={{ myProp: myValue }} /> ' +
+        'with <AppContainer><App myProp={myValue} /></AppContainer>.'
       );
     }
+
+    return undefined;
   },
-  children: function (props) {
+  children(props) {
     if (React.Children.count(props.children) !== 1) {
-      return new Error(`Invalid prop "children" supplied to AppContainer. Expected a single React element with your app’s root component, e.g. <App />.`);
+      return new Error('Invalid prop "children" supplied to AppContainer.' +
```

Comment:
There’s a missing space after the period in the error message. The sentences will collide.
Also a style nit: let’s put a newline right after the opening paren.